app/views/govuk_publishing_components/components/docs/heading.yml

```diff
@@ -21,6 +21,12 @@ examples:
     data:
       text: 'Original consultation'
       heading_level: 3
+  different_font_sizes:
+    description: Choose a font size appropriate to the chosen heading level. Valid options are 2 (24px) and 3 (19px), with the component defaulting to 27px, appropriate for a H1. This option is not tied to the heading_level option in order to give flexibility.
```

Comment:
I'm not sure I understand the logic behind using the numbering system here. It seems to suggest that all h2s and h3s ideally should be 24px and 19px respectively (I wasn't aware that was the case)... but also suggests they're not enforced so we don't have to do that?

Reply:
They're not, I just chose some semi-sensible numbers for the different heading sizes. We don't have a convention (our heading sizes are all over the place).

Reply:
I like 24 and 19, much better, will change, thanks.